docs/maya/activation.md

```diff
@@ -0,0 +1,78 @@
+2. Set the desired *Operator*.
+3. Type in a fixed value in *Value*, animated it or connect the plug with an AdonisFX sensor's output.
+
+> [!NOTE]
+> All operators will be evaluated from top to bottom (starting from index 0 and ending on index N).
+
+## Operators
+
+1. **Over (Override)**: Overrides the accumulated output activation value with Value. If the current accumulated activation value is 1.0 and Value is 2.0 the output activation will be 2.0.
+2. **Add (Add)**: Adds the accumulated output activation value with Value. If the current accumulated activation value is 1.0 and Value is 2.0 the output activation will be 3.0.
+3. **Sub (Subtract)**: Subtracts the accumulated output activation value with Value. If the current accumulated activation value is 1.0 and Value is 2.0 the output activation will be 0.0.
+4. **Mult (Multiply)**: Multiplies the accumulated output activation value with Value. If the current accumulated activation value is 1.0 and Value is 2.0 the output activation will be 2.0.
+5. **Div (Divide)**: Dives the accumulated output activation value with Value. If the current accumulated activation value is 1.0 and Value is 2.0 the output activation will be 0.5.
+
+## Example
+
+<figure markdown>
+  ![AdnActivation example setup](images/adn_activation_example.png)
+  <figcaption><b>Figure 1</b>: Closeup use case in which AdnActivation is created for merging two activations to drive a muscle's activation.</figcaption>
+</figure>
+
+In the above setup we have the following characteristics:
+
+1. 1 AdnSensorRotation.
+2. 1 AdnSensorPosition.
+3. 1 AdnActivation.
+4. 1 AdnMuscle.
+5. 2 Inputs added to AdnActivation.
+6. Input 1 has an Over operator and is connected to the AdnSensorRotation.
+7. Input 2 has an Add operator and is connected to the AdnSensorPosition.
+8. The bypass option is unchecked for both inputs.
+9. Out Value will be: AdnSensorRotation Activation + AdnSensorPosition Activation.
+
+## Attributes
+
+### Inputs Attributes
+The *inputs* attribute is presented as an array of 3 attributes which can be found below.
+
+| Name | Type | Default | Animatable | Description |
+| :--- | :--- | :------ | :--------- | :---------- |
+| **Bypass Operator** | Boolean | True            | ✓ | If enabled, it bypasses the current operator in the input list, which will not contribute to the final activation value |
+| **Value**           | Float   | 0.0             | ✓ | Activation value that will contribute, given the operator type, to the final activation. |
+| **Operator**        | Enum    | 0 (Over)        | ✓ | Operator used to contribute to the final activation. This can be: (0) Over; (1) Add; (2) Sub; (3) Mult; (4) Divide. |
+
+### Output Attributes
+| Name | Type | Default | Animatable | Description |
+| :--- | :--- | :------ | :--------- | :---------- |
+| **Output Min** | Float | 0.0 | ✓ | Minimum supported output activation value. |
+| **Output Max** | Float | 1.0 | ✓ | Maximum supported output activation value. |
```

Comment:
Could you add the range (if any), I don't remember

Reply:
Added. I did not add them because they are soft. But I've added a clarification